src/components/notifications-list-view.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import axios from 'axios'
+import path from 'path'
+import { Link } from 'react-router-dom'
+import { Dropdown, Header, Image } from 'semantic-ui-react'
+
+import { urlUserNotifications, urlAllNotifications } from '../urls'
+import '../css/inline.css'
+
+class NotificationsListView extends React.Component {
+  state = {
+    notifications: [],
+    count: 0
+  }
+  getNotifications = () => {
+    axios
+      .get(urlUserNotifications())
+      .then(res => {
+        this.setState({
+          notifications: res.data['results'],
+          count: res.data['count']
+        })
+      })
+      .catch(err => {
+        console.error(err) // TODO: Remove console.log, use error handlers
```

Comment:
We should remove the console and left them as todos only, or we may merge this after handling the errors using newly added toast package.